tests/cases/folder_test.py

```diff
@@ -38,7 +38,7 @@ def tearDownModule():
 
 class FolderTestCase(base.TestCase):
     def setUp(self):
-        events.unbindAll()
```

Comment:
Note, adding the comment is a quick hack. I'll actually remove the line once this is settled.